src/components/gui/gui.jsx

```diff
@@ -323,6 +331,9 @@ const GUIComponent = props => {
 GUIComponent.propTypes = {
     accountNavOpen: PropTypes.bool,
     activeTabIndex: PropTypes.number,
+    authorId: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
```

Comment:
When will these be `bool`s? Why not always an (optional) string? If it's not supplied it would be `undefined` which isn't a `bool`.

Reply:
So, the reason is that www's preview.jsx is doing logic like:

```
    const authorId = authorPresent && author.id && author.id.toString();
```

if authorPresent is `false`, the value of authorId will be `false`, not `undefined`.

I could set this differently -- as a ternary, something like

```
    const authorId = authorPresent && author.id && author.id.toString() ? author.id.toString() : undefined;
```
...but my understanding is that it's an anti-pattern to set variables to `undefined` this way?

Reply:
Done